roles/openshift_logging/tasks/generate_certs.yaml

```diff
@@ -0,0 +1,132 @@
+---
+# we will ensure our secrets and configmaps are set up here first
+- name: Checking for ca.key
+  stat: path="{{generated_certs_dir}}/ca.key"
+  register: ca_key_file
+  check_mode: no
```

Comment:
Yes, the issue that I came across when testing with the check_mode flag was that variables were still evaluated even if the task wasn't going to be run. So in this case where we are registering `ca_key_file`, if this is skipped then the tasks that follow and use this fact will fail because it isn't defined...